app/templates/partials/answer.html

```diff
@@ -1,7 +1,8 @@
 {% import 'macros/helpers.html' as helpers %}
 
-<div class="answer answer--{{answer.schema_item.type|lower}}" id="{{answer.id}}">
-  {% set invalid = answer.is_valid == False %}
+{% set invalid = answer.is_valid == False %}
+
+<div class="answer answer--{{answer.schema_item.type|lower}} {{'js-has-errors' if invalid}}" id="{{answer.id}}">
```

Comment:
The `answer.id` here is not unique when we add new answers. This needs to be fixed at a high level (not in the template) as that id is then used to generate id's further down the tree.